resources/config/varnish/fos_purge.vcl

```diff
@@ -12,6 +12,17 @@ sub fos_purge_recv {
         if (!client.ip ~ invalidators) {
             return (synth(405, "Not allowed"));
         }
+
+        # If http header "xkey" is set, we assume purge is on key and you have Varnish xkey installed
+        if (req.http.xkey) {
+            # By default we recommend using soft purge to respect grace time, if you need to hard purge use:
+            # set req.http.n-gone = xkey.purge(req.http.xkey);
+            set req.http.n-gone = xkey.softpurge(req.http.xkey);
```

Comment:
I can add it here at the top, but 1: doesn't it need to be at the top of the file? And 2: adding it by default will make it break if you don't have it afaik.